crates/lib/src/db/def.rs

```diff
@@ -300,14 +292,13 @@ impl From<FieldDef<'_>> for ProductTypeElement {
 
 /// A struct representing the definition of a database column.
 #[derive(Debug, Clone, Eq, PartialEq, PartialOrd, Ord, ser::Serialize, de::Deserialize)]
-#[sats(crate = crate)]
 pub struct ColumnDef {
     pub col_name: Box<str>,
     pub col_type: AlgebraicType,
 }
 
-impl From<ProductType> for Vec<ColumnDef> {
-    fn from(value: ProductType) -> Self {
+impl ColumnDef {
+    pub fn from_product_type(value: ProductType) -> Vec<ColumnDef> {
```

Comment:
This change was necessary due to the orphan rule (you can't impl From<T> for Vec<StructDefinedInThisCrate>)